tests/lib/test_constructor.py

```diff
@@ -41,7 +41,20 @@ def construct1(constructor, node):
     def represent1(representer, native):
         return representer.represent_mapping("!tag1", native.__dict__)
 
+    def my_time_constructor(constructor, node):
+        seq = constructor.construct_sequence(node)
+        dt = seq[0]
+        tz = None
+        try:
+            tz = dt.tzinfo.tzname(dt)
+            tz = 'UTC' + tz
```

Comment:
Hmm, does the default `datetime.timezone` implementation include the `UTC` prefix by default for python 3? Changing the `__str__` for the `timezone` helper for python 2 to include `UTC` here might make more sense to keep those aligned.

Reply:
I was wondering about that, too.
Since we expect it to be UTC (plus offset) always I guess I can add that.